kernel/configs/arm64_kata_kvm_4.19.x

```diff
@@ -1533,7 +1576,26 @@ CONFIG_POWER_SUPPLY=y
 # CONFIG_BATTERY_BQ27XXX is not set
 # CONFIG_CHARGER_MAX8903 is not set
 # CONFIG_HWMON is not set
-# CONFIG_THERMAL is not set
+CONFIG_THERMAL=y
+# CONFIG_THERMAL_STATISTICS is not set
+CONFIG_THERMAL_EMERGENCY_POWEROFF_DELAY_MS=0
+CONFIG_THERMAL_OF=y
+# CONFIG_THERMAL_WRITABLE_TRIPS is not set
+# CONFIG_THERMAL_DEFAULT_GOV_STEP_WISE is not set
+# CONFIG_THERMAL_DEFAULT_GOV_FAIR_SHARE is not set
+CONFIG_THERMAL_DEFAULT_GOV_USER_SPACE=y
+# CONFIG_THERMAL_DEFAULT_GOV_POWER_ALLOCATOR is not set
+# CONFIG_THERMAL_GOV_FAIR_SHARE is not set
+# CONFIG_THERMAL_GOV_STEP_WISE is not set
+# CONFIG_THERMAL_GOV_BANG_BANG is not set
+CONFIG_THERMAL_GOV_USER_SPACE=y
```

Comment:
On purpose? Maybe you gained some default `=y` items during a new config generation. If you have not already, you might want to scrutinize the diff for any stray new additions you didn't intend.

Reply:
noooo, my bad, I just quickly compared the new diff with x86 config(`x86_64_kata_kvm_4.19.X`) to make two configs as same as possible, not scrutinizing what these diff represent.
I will turn off the CONFIG_THERMAL, since VM may not be involved in thermal managment. ;)